examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -45,13 +58,13 @@ func (o *Ownable) DropOwnership() error {
 }
 
 // CallerIsOwner checks if the caller of the function is the Realm's owner
-func (o *Ownable) CallerIsOwner() error {
+func (o *DefaultOwnable) CallerIsOwner() error {
```

Comment:
```suggestion
func PrevRealmIsOwner(o Ownable) error {
```

Switch to prevRealm instead of origCaller in the check.

Make it a package helper instead of a method to keep the interfaces small.

Reply:
https://github.com/gnolang/gno/pull/2198/commits/b206494583e6bcb6bc2efdf1ce7df76516833f93

Reply:
> Switch to prevRealm instead of origCaller in the check.
> 
> Make it a package helper instead of a method to keep the interfaces small.

I have created a separate 'helper' package to contain this validation